.github/workflows/sdk.yml

```diff
@@ -343,9 +342,43 @@ jobs:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
 
+  go-sdk:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/4paradigm/hybridsql:latest
```

Comment:
the job only need compile openmldb. create an environment variable `OPENMLDB_BUILD_TARGET` and set to `openmldb`